Trilinos fails

```
[trilinos] Processing enabled package: Belos (Libs)
[trilinos] Processing enabled package: Anasazi (Libs)
[trilinos] Processing enabled package: Stratimikos (Libs)
[trilinos] Processing enabled package: NOX (Libs)
[trilinos] Processing enabled package: PyTrilinos (Libs)
[trilinos] -- Found PythonInterp: /local/certik/bld/python/3hzpjjkvwkn3/bin/python (Required is at least version "2.2") 
[trilinos] -- Python version 2.7.8
[trilinos] -- Found PythonLibs: /local/certik/bld/python/3hzpjjkvwkn3/lib/libpython2.7.so (found suitable version "2.7.8", minimum required is "2.2") 
[trilinos] -- Found NumPy: 1.8.2 (Required is at least version "1.0") 
[trilinos] CMake Error at packages/PyTrilinos/CMakeLists.txt:170 (MESSAGE):
[trilinos]   SWIG version 3.0.2 is greater than maximum version 2.0.8
[trilinos] 
[trilinos] 
[trilinos] -- Configuring incomplete, errors occurred!
[trilinos] See also "/local/certik/tmp/trilinos-esdpedkx4qdu/_build/CMakeFiles/CMakeOutput.log".
[trilinos] See also "/local/certik/tmp/trilinos-esdpedkx4qdu/_build/CMakeFiles/CMakeError.log".
[trilinos|ERROR] Command '[u'/bin/bash', '_hashdist/build.sh']' returned non-zero exit status 1
[trilinos|ERROR] command failed (code=1); raising
```
Fix on the way.

--- a/pkgs/trilinos.yaml
+++ b/pkgs/trilinos.yaml
@@ -3,8 +3,8 @@ dependencies:
   build: [python, swig, mpi, numpy, openblas, suitesparse]
 
 sources:
-- key: tar.bz2:2orzw6wvyyor4fat7hsvb4fidazkg3sk
-  url: http://trilinos.sandia.gov/download/files/trilinos-11.10.2-Source.tar.bz2
+- key: tar.bz2:uqktsqkffhdfsbjgbm567y5o4ty52eav
+  url: http://trilinos.org/oldsite/download/files/trilinos-11.12.1-Source.tar.bz2
 
 defaults:
   # include/trilinos/Makefile.export.Trilinos contains hard-coded path

